cluster-support-bot.py

```diff
@@ -92,6 +108,15 @@ def _handle_message(payload):
             logger.error(response)
 
 
+def handle_uuid_mention(text):
+    regex = re.compile('.*([a-f0-9]{8}-?[a-f0-9]{4}-?4[a-f0-9]{3}-?[89ab][a-f0-9]{3}-?[a-f0-9]{12}).*', re.I)
```

Comment:
This compiles the regexp for each received message.  I'd rather have a global `uuid_regexp` defined right after `bot_mention` up at the top of the file.

Reply:
good call. :)